src/init-ceph.in

```diff
@@ -39,12 +39,12 @@ else
 fi
 
 if [ -n "$CEPH_BIN" ] && [ -n "$CEPH_ROOT" ] && [ -n "$CEPH_BUILD_DIR" ]; then
-  BINDIR=$CEPH_BIN
-  SBINDIR=$CEPH_ROOT/src
-  ETCDIR=$CEPH_BIN
-  LIBEXECDIR=$CEPH_ROOT/src
-  SYSTEMD_RUN=""
-  ASSUME_DEV=1
+    BINDIR=$CEPH_BIN
```

Comment:
@odivlad in general, we don't do massive reformatting in a single commit. this makes it hard to "git blame" a file. instead, it would be better to tweak the formatting of the code you are changing.